src/PKSim.Core/Model/ParameterExtensions.cs

```diff
@@ -105,13 +105,7 @@ public static bool ValueDiffersFromDefault(this IParameter parameter)
          return !ValueComparer.AreValuesEqual(parameter.Value, parameter.DefaultValue.Value, CoreConstants.DOUBLE_RELATIVE_EPSILON);
       }
 
-      //TODO Use ValueOrigin state when implemented. It should be != than PKSim default and probably mereged with method above
-      // This is just a dymmy method for now to satisfy requalification project
-      public static bool ParameterHasChanged(this IParameter parameter)
-      {
-         var canBeEdited = parameter.Visible && parameter.Formula.IsConstant();
-         return parameter.ValueDiffersFromDefault() || canBeEdited && parameter.Value != 0;
-      }
+      public static bool ParameterHasChanged(this IParameter parameter) => parameter.Visible && parameter.IsDefault == false;
```

Comment:
This is really the one difference. Only take parameter that are visible at the moment. Not sure if this is required or not

Reply:
I would remove `parameter.Visible` check.
E.g. parameters hidden in PK-Sim could be changed in MoBi; after reimport of a simulation back in PK-Sim those changes should appear under changed parameters